packages/playground/css/__tests__/css.spec.ts

```diff
@@ -405,3 +405,13 @@ test("relative path rewritten in Less's data-uri", async () => {
     /^url\("data:image\/svg\+xml,%3Csvg/
   )
 })
+
+test('PostCSS source.input.from includes query', async () => {
+  const code = await page.textContent('.postcss-source-input')
+  // should resolve assets
+  expect(code).toContain(
+    isBuild
+      ? '/postcss-source-input.css?used&query=foo'
```

Comment:
Where does the `used` come from?

Reply:
I comes from this one.
https://github.com/vitejs/vite/blob/1d468c8e6f02b0d0e362aa2d6542af1e1f55ab45/packages/vite/src/node/plugins/importAnalysisBuild.ts#L187-L190

Reply:
Interesting! Thanks for the link 🙂